Cargo.toml

```diff
@@ -30,7 +30,7 @@ default = ["default-tls"]
 
 # Note: this doesn't enable the 'native-tls' feature, which adds specific
 # functionality for it.
-default-tls = ["hyper-tls", "native-tls-crate", "__tls", "tokio-native-tls"]
+default-tls = ["hyper-tls", "native-tls-crate", "native-tls-crate/alpn", "__tls", "tokio-native-tls"]
```

Comment:
I think enabling this by default could be a breaking change, because it will try to access new symbols in the linked libraries that may not exist. I wish the feature could be made to check at runtime, but it currently doesn't.

So, we'd probably need a separate `native-tls-alpn` feature in reqwest. Unless there's a better option?